src/ui/atoms/listItem/ListItem.tsx

```diff
@@ -34,12 +34,14 @@ export type ListItemProps = Pick<ListProps, 'layout'> & {
   readonly onClick?: () => void
 }
 
-type TitleProps = Pick<ListItemProps, 'title'>
-type DescriptionProps = Pick<ListItemProps, 'description'>
+type TitleProps = {
```

Comment:
I think we should create a generic type for `string | JSX.Element` because we use it very often. 
We could put it in `superTypes.ts`, what do you think @fredericvilcot ?

Reply:
Ok for a generic type if we can find a good name for it (and you know, naming things is hard). But don't put it in the superTypes file because it should not be tainted with UI stuffs (look at the position of the file in the hierarchy)

Reply:
Yes you're right! Each time I have the impression that it is under `UI`folder!
We can consider the creation of a `types.ts` file under `UI`.

Reply:
Sounds good to me mind making an issue for this (out of scope of this PR)? @lolottetheclash 

@RaphaelEscrig I don't really understand why you are separating the types? For me the types are dependent on the types of ListItemProps

Reply:
@ErikssonJoakim for me we can create a type in a file and apply it to `ListItem`, the idea is not to apply the changes to all other files. The refactor will be done in another specific PR.

Reply:
@ErikssonJoakim, I did it because with the `TitleProps = Pick<ListItemProps, 'title'>`, title could be `string | JSX.Element | undefined`. It wasn't correct because we render this component only if title isn't undefined (new conditional rendering on the `<ListItem />` component) so the props received on the <Title /> could never be undefined.

Reply:
TypeScript provides a high level typing to reverse optional to required:

```typescript
TitleProps = Required <Pick<ListItemProps, 'title'>>
```

Reply:
simple and elegant solution!